docs/std-seq.md

```diff
@@ -99,3 +99,29 @@ Usage:
 | example | equals |
 |:-|:-|
 | `//seq.repeat(2, "hots")` | `"hotshots"` |
+
+## `//seq.trim_prefix(prefix <: array, subject <: array) <: array` <br/> `//seq.trim_prefix(prefix <: string, subject <: string) <: string`
```

Comment:
what about `Bytes`? the same with `trim_suffix`

Reply:
Good point. I copy-pasted another section that is also missing bytes.